amida-tech/indaba-client
# INBA-737/ remove search grommet

**Note** - Do not merge until css has been fixed, Lacy will do after code is checked 

If this PR fixes a bug, you _must_ add test cases representative of the bug.

Please refer to [Tettra](https://app.tettra.co/teams/amida/pages/amida-pull-request-and-code-review-guide) for PR review guidelines.

#### What does this PR do?
Removes the Grommet Search and creates a custom component called Search 

#### Related JIRA tickets:
https://jira.amida-tech.com/browse/INBA-737

#### How should this be manually tested?
Click on Search, dropdown appears 
Click on Search Again, dropdown disappears
Click on Search, dropdown appears 
Click away from search (anywhere else on page) and dropdown disappears 
Search for user, should find user. If you click on user, user popup appears 

#### Background/Context
grommet had an error/ did not like using it. So we removed it. 

#### Screenshots (if appropriate):
